### PR TITLE
Import torch-onnx-utils

### PR DESCRIPTION
**Pull request by take-cheeze**
- ONNX utilties to implement some features in onnx-chainer
- Exporting test cases and other useful tools!

### CODE REVIEW

**Comment by take-cheeze** [comment]
Seems to fail with test... I'll fix it...

**Comment by take-cheeze** [comment]
Finally test passed!

File: docs/tou.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# PyTorch ONNX utils
+ONNX utilties to implement features in onnx-chainer
+
+## Requirements
+Install onnx package as an additional requirements to use this package
+
+## API
+- `tou.export_testcase`
+    - ONNX Exportor with testcase outputs like ONNX testcases
```
Comment (kmaehashi) [inline docs/tou.md:9]:
```suggestion
    - ONNX Exportor with testcase outputs like [ONNX testcases](https://github.com/onnx/onnx/tree/master/onnx/backend/test/data/node)
```

File: pytorch_pfn_extras/tou/export_testcase.py [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+        if is_large_tensor(init, large_tensor_threshold):
+            _strip_raw_data(init)
+
+    # Overwrite onnx file
+    with open(onnx_path, "wb") as model_file:
+        model_file.write(model.SerializeToString())
+
+
+def export_testcase(
+        model, args, out_dir, output_grad=False, metadata=True,
```
Comment (kmaehashi) [inline pytorch_pfn_extras/tou/export_testcase.py:107]: Maybe this is better later enhancement?
```suggestion
        model, args, out_dir, *, output_grad=False, metadata=True,
```

File: docs/tou.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# PyTorch ONNX utils
+ONNX utilties to implement features in onnx-chainer
+
+## Requirements
+Install onnx package as an additional requirements to use this package
+
+## API
+- `tou.export_testcase`
```
[the remark this answers is not in the document]
Comment (take-cheeze) [reply docs/tou.md]: There is more detailed document so I'll add it separate PR.
That should cover annotate API too

File: tests/pytorch_pfn_extras_tests/onnx/test_annotate.py [hunk 4]
```diff
@@ -0,0 +1,317 @@
+from collections import OrderedDict
+from contextlib import nullcontext
```
Comment (take-cheeze) [inline tests/pytorch_pfn_extras_tests/onnx/test_annotate.py:2]: nullcontext is from 3.7 https://docs.python.org/3/library/contextlib.html#contextlib.nullcontext

**Review by kmaehashi — APPROVED** [review]
LGTM!

**Pull request merged**